content/en/docs/concepts/containers/images.md

```diff
@@ -283,43 +283,13 @@ kubectl create secret docker-registry myregistrykey --docker-server=DOCKER_REGIS
 secret/myregistrykey created.
 ```
 
-If you need access to multiple registries, you can create one secret for each registry.
-Kubelet will merge any `imagePullSecrets` into a single virtual `.docker/config.json`
-when pulling images for your Pods.
+If you already have a Docker credentials file, you can import this instead,
```

Comment:
This isn't quite a complete sentence. Can you please explain how someone could import a Docker credentials file?

Reply:
Oops, wrong punctuation.

Reply:
Now weworded.